neqo-transport/tests/server.rs

```diff
@@ -56,6 +56,47 @@ pub fn complete_connection(
     connected_server(server)
 }
 
+#[test]
+fn connect_to_nothing() {
+    // Start a server to get the token that will be used for calling
+    // `client.enable_resumption`.
+    let mut server = default_server();
+    let token = generate_ticket(&mut server);
+
+    // Discharge the old connection so that we don't have to worry about it.
+    let mut now = now();
+    let t = server.process(None, now).callback();
+    now += t;
```

Comment:
Do you really need this given that the code below doesn't even use the server?

Reply:
Yes, I think we need it because this seems to be the only to get the transport parameters from peer.

Reply:
The client should be remembering transport parameters through the session ticket (token) that it gets above.